### DISCUSSION TITLE
MultiSelectListBox for multiple book genre

### DISCUSSION THREAD

**Discussion by kumawatanupriya** [Project-Books/book-project#629] [Q&A] [answered]
Hi,
I am working on an issue to support multiple genre for a books, I am really new to vaadin framework. Currently Genre is using ComboBox, I changed it to MultiSelectListBox. I am able to select multiple genre in the form but the UI is little distorted.  

Attaching a screenshot, I am not able to figure out what's wrong. 
<img width="1057" alt="Screenshot 2021-02-19 at 11 02 46 PM" src="https://user-images.githubusercontent.com/32339660/108539959-a2027480-7306-11eb-9d09-c909fadc725f.png">
@knjk04

**Comment by knjk04** [comment, marked as answer]
@kumawatanupriya So sorry about this, the issue wasn't clear. This was originally a Vaadin issue, but I removed the tag because we're migrating away from it, so we don't want to be adding new features to it. This issue is now a backend-only issue. Once we get that working, I'll make a new frontend React issue for it.

Also, I forgot to mention before that this will be going into the `0.2.0` branch (and release), so please create your branch from `0.2.0`? When you're ready to create a pull request, please set the destination branch to `0.2.0`.

Hope that helps and I'm very sorry about the inconvenience! I've updated the issue description accordingly.
